src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -291,7 +292,11 @@ public OnSucessActionListener(RestChannel channel) {
 
 		@Override
 		public final void onFailure(Exception e) {
-			internalErrorResponse(channel, "Error "+e.getMessage());
+			if (e instanceof VersionConflictEngineException) {
```

Comment:
`ExceptionsHelper.unwrapCause(e)` (sorry, missed it in the last review)

Reply:
Done.